src/profile/__tests__/profile.hook.test.tsx

```diff
@@ -231,7 +231,7 @@ describe('Profile.ts useProfile hook ', () => {
       });
       apiTokenActions.fetch(createApiTokenFetchPayload());
       await waitFor(() => expect(apiTokenActions.getStatus()).toBe('loaded'));
-      profileActions.fetch();
+      profileActions.request({});
```

Comment:
Finally fixed. The issue was with clearing user and calling fetch correct times. In some cases fetch() was called manually AND auto fetch was triggered too. The mocked api response is set to respond once and when called twice, the second one will return an error. Tests failed randomly, because sometimes both requests finished before expect() was called and sometimes only first request finished.